rulesets/google-ua-enhanced-ecommerce.js

```diff
@@ -1,325 +1,156 @@
-window['_dlo_rules_google_enhanced_ecommerce'] = [
+window['_dlo_rules_google_ec'] = [
   {
-    "id": "fs-ua-pageview",
-    "source": "dataLayer",
+    "id": "fs-ga-pageview", "source": "dataLayer",
     "operators": [
-      {
-        "name": "query",
-        "select": "$[?(pageType, pageName)]"
-      },
-      {
-        "name": "insert",
-        "value": "pageview"
-      }
+      { "name": "query", "select": "$[?(pageType, pageName)]" },
+      { "name": "insert", "value": "pageview" }
     ],
     "destination": "FS.event"
   },
   {
-    "id": "fs-ua-e-commerce-detail-action",
+    "id": "fs-ga-e-commerce-detail-action",
```

Comment:
does the name of the file still relevant if we're removing all the ua references?